app/pages/certification-redirect.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {Component} from 'react';
+import {Row, Col, Button, Form} from 'react-bootstrap';
+import {graphql} from 'react-relay';
+import moment from 'moment';
+import {certificationRedirectQueryResponse} from 'certificationRedirectQuery.graphql';
+import DefaultLayout from '../layouts/default-layout';
+
+interface Props {
```

Comment:
This should extend the `CiipPageComponentProps`, which defines the type of `router` for you